docs/man/man1/ansible-galaxy.1.asciidoc.in

```diff
@@ -145,6 +144,203 @@ The path to the directory containing your roles. The default is the *roles_path*
 configured in your *ansible.cfg* file (/etc/ansible/roles if not configured)
 
 
+SEARCH
+------
+
+The *search* sub-command returns a filtered list of roles found at
+galaxy.ansible.com.
```

Comment:
not sure if its worth changing this to 'the api server' for when people are able to have private ones.